### ISSUE TITLE
Service Worker support

### ISSUE THREAD

**Issue by staminna** [HubSpot/offline#239]
Hi

I am using [Google's](https://googlechrome.github.io/samples/service-worker/read-through-caching/index.html) [Read Through Service](https://github.com/GoogleChrome/samples/tree/gh-pages/service-worker) worker example, although it works fine without Service Worker's code, offline.js doesn't detect when the connection is lost and SW code kicks in.

**Comment by dupenna** [comment]
Hi. 

I'm experiencing the same issue with service worker.

It works fine when I load it in online mode, but if I go offline and reload the URL, the first status in "on".

@staminna, did you found a work around?

**Comment by staminna** [comment]
Hi

No. I’m not into using a project when they take months to reply or don’t help at all.
I have probably used some other solution.

Sorry

**Comment by ArmorDarks** [comment]
Obviously, you should filter in Service Worker fetch to the URL, which you're hitting with Offline, otherwise, SW will return cached reply and Offline will think that everything is ok.
